thorlcr/thorutil/thbuf.cpp

```diff
@@ -2008,15 +2033,15 @@ class CSharedFullSpillingWriteAhead : public CInterfaceOf<ISharedRowStreamReader
             {
                 StringBuffer tracing;
                 getFileIOStats(tracing, iFileIO);
-                activity.ActPrintLog("CSharedFullSpillingWriteAhead: removing spill file: %s%s", iFile->queryFilename(), tracing.str());
+                activity.ActPrintLog("CSharedFullSpillingWriteAhead: removing spill file: %s%s", tempFileOwner->queryIFile().queryFilename(), tracing.str());
                 closeWriter();
-                iFile->remove();
+                tempFileOwner.clear();
```

Comment:
what recreate it? I think as it stands, it would crash after reset if instance is reused.
Does it need to be cleared?

Reply:
I did see that reset wouldn't work.  However, it looks like the behaviour is exactly same as before and the previous implementation would have crashed as well.  If this isn't a new bug introduced by this change, I can create a jira to address the issue.   Is this an existing bug? @jakesmith